packages/@aws-cdk/cdk/lib/core/construct.ts

```diff
@@ -35,9 +35,10 @@ export class Construct {
      * Creates a new construct node.
      *
      * @param parent The parent construct
-     * @param props  Properties for this construct
+     * @param name The logical identfifier of the new Construct
+     * @param props Optional properties for this construct
      */
-    constructor(parent: Construct, name: string) {
```

Comment:
Please revert this change to `Construct`

Reply:
As said above, not needed anymore.